src/main/java/org/folio/services/voucher/BatchVoucherGenerateService.java

```diff
@@ -73,7 +74,10 @@ public Future<BatchVoucher> buildBatchVoucherObject(BatchVoucherExport batchVouc
     return voucherService.getVouchers(voucherCQL, 0, Integer.MAX_VALUE, requestContext)
       .compose(vouchers -> {
         if (vouchers.getVouchers().isEmpty()) {
-          throw new HttpException(404, "Vouchers for batch voucher export were not found");
+          var param = new Parameter().withKey("voucherCQL").withValue(voucherCQL);
+          var error = new Error().withMessage("Vouchers for batch voucher export were not found").withParameters(List.of(param));
```

Comment:
There is three different message that are similar thinking to skip to introduce error code